Code/Framework/AzCore/Platform/Common/WinAPI/AzCore/Utils/Utils_WinAPI.cpp

```diff
@@ -29,7 +30,8 @@ namespace AZ
             result.m_pathIncludesFilename = true;
             // Platform specific get exe path: http://stackoverflow.com/a/1024937
             // https://docs.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-getmodulefilenamea
-            const DWORD pathLen = GetModuleFileNameA(nullptr, exeStorageBuffer, static_cast<DWORD>(exeStorageSize));
+            wchar_t pathBufferW[AZ::IO::MaxPathLength] = { 0 };
```

Comment:
nit: This likely isn't a huge deal but I'm pretty sure this will initialize the whole buffer to zero where you could just do `pathBufferW[0]= '\0';` to save having to zero the whole buffer (if it's not a hotspot might not be worth it but I remember a mail from years ago by Patrick Wyatt about this iirc and there is definitely a measurable difference)

Reply:
sure, will change